src/repos.ts

```diff
@@ -34,50 +142,32 @@ export async function syncRepoSettings() {
       });
   });
   await Promise.all(ps);
+}
 
-  console.log('Updating master branch protection...');
-
-  // The Go repositories are synchronized with a different git
-  // repository, so direct commits need to be made.
-  const ignoreRepos = [
-    'googleapis/google-cloud-go',
-    'googleapis/google-api-go-client',
-  ];
+const languageConfig: LanguageConfig = {
+  nodejs: {
+    requiredStatusChecks: [
+      'ci/kokoro: Samples test',
+      'ci/kokoro: System test',
+      'ci/kokoro: Windows',
+      'ci/kokoro: docs',
+      'ci/kokoro: lint',
+      'ci/kokoro: node8',
+      'ci/kokoro: node10',
+      'ci/kokoro: node12',
+      'cla/google',
+    ],
+    ignoredRepos: [
+      'GoogleCloudPlatform/nodejs-docs-samples',
+      'GoogleCloudPlatform/getting-started-nodejs',
+      'gapic-generator-typescript',
```

Comment:
It did not ignore it :) I fixed it manually, but, I'm guessing, we need a full name here.

```suggestion
      'googleapis/gapic-generator-typescript',
```

Reply:
lol oops.  Sorry about that!